kbfscrypto/auth_token.go

```diff
@@ -11,7 +11,7 @@ import (
 	"time"
 
 	"github.com/keybase/client/go/auth"
-	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/kbun"
```

Comment:
I'd prefer if this package was called `kbusernames` or `kbusers`. `kbun` doesn't actually tell me anything.

Reply:
Added a kbname alias everywhere -- this package will become kbname upstream.